docs/usage_examples.rst

```diff
@@ -369,6 +369,27 @@ This obviously is not an efficient query for large data sets, but it can be conv
 Joins
 -----
 
+Json Fields
+-----------
+Filtering and ordering by json fields is currently in an experimental phase.
+
+Queryset example:
+
+.. code-block:: python
```

Comment:
You'll need a blank line between the `code-block` and the code. Currently, it won't show up https://github.com/wesokes/django-query-builder/blob/a2080d1560cdeeed2a41ee5f0652f4fde626bf59/docs/usage_examples.rst#json-fields